packages/typescript/chat-core/test/ai/ai-provider.test.tsx

```
@@ -0,0 +1,172 @@
+// AI Provider tests
+import React from 'react';
+import { describe, test, expect } from 'bun:test';
+import { AIProvider, useAIConfig } from '../../src/ai/providers/ai-provider';
```

ESLint

[error] 'useAIConfig' is defined but never used.

(@typescript-eslint/no-unused-vars)